packages/cli/src/commands/dev.js

```diff
@@ -19,7 +19,7 @@ export const builder = (yargs) => {
       description: 'Which dev server(s) to start',
       type: 'array',
     })
-    .positional('forward', {
+    .option('forward', {
```

Comment:
before I merge, can we think of any _good_ reason why this would have been a positional originally?

Reply:
Just a typo/bug - we probably never noticed it because it runs headless in the workflow